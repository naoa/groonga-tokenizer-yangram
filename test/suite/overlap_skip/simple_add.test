register tokenizers/yangram

yangram_register 2 --overlap_skip 1
tokenize TokenYaBigramSkipOverlap "今日は雨だな" NormalizerAuto --mode ADD
