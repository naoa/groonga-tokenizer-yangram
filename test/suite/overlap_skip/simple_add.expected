register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register 2 --overlap_skip 1
[[0,0.0,0.0],"TokenYaBigramSkipOverlap"]
tokenize TokenYaBigramSkipOverlap "今日は雨だな" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "日は",
      "position": 1
    },
    {
      "value": "は雨",
      "position": 2
    },
    {
      "value": "雨だ",
      "position": 3
    },
    {
      "value": "だな",
      "position": 4
    },
    {
      "value": "な",
      "position": 5
    }
  ]
]
